docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,200 @@
+:toc:
+:toclevels: 3
+
+= Documentation for core maintainers
+
+toc::[]
+
+Jenkins core is a central component of the project which serves millions of users, and it is critical to maintain it in a good shape.
+We want to ensure quality of the integrated changes and continuity of the project,
+and hence the Jenkins core pull request review and merge process is more sophisticated than for the majority of plugins.
+
+== Scope of the document
+
+This document applies to the following components:
+
+* Jenkins core
+* Jenkins modules
+* Libraries included into the Jenkins core
+* Core components like Winstone, Executable WAR, etc.
+
+Jenkins Remoting library is not in the scope for this document,
+because it is handled as a sub-project with independent maintainer team.
+Remoting updates in the core are subject to the process though.
+
+== Team
+
+=== Communication channels
+
+* link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer Mailing List]
+* Chat: N/A, to be created
+* Office hours: N/A, to be created
+
+=== Roles
+
+* Contributor
+* Core Pull Request Reviewer
+* Core Maintainer
+* Release Team Member
+
+**Contributors** submit pull requests to the Jenkins core and review changes submitted by others.
+They can also review and triage incoming issues in Jenkins Jira.
+There is no special preconditions to do so, anyone is welcome to contribute.
+
+**Core Pull Request Reviewers** is the next step for contributors who are interested to review Jenkins pull requests on a regular basis.
+They get [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
```

Comment:
```suggestion
They get https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization[Triage permissions] so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
```